src/main/resources/view/WarningWindow.fxml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.net.URL?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.Scene?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.image.Image?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.stage.Stage?>
+
+<fx:root resizable="false" title="Help" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
+    <icons>
+        <Image url="@/images/help_icon.png" />
+    </icons>
+    <scene>
+        <Scene>
+            <stylesheets>
+                <URL value="@HelpWindow.css" />
```

Comment:
I dont know if its my issue or the .css is excluded in the gitignore.
But the style does not look consistent with the help window on my end.
![image](https://github.com/AY2324S2-CS2103T-T11-4/tp/assets/35594369/251f25d0-4f28-48a5-a482-655ebc97ee07)

Reply:
Hm looks okay on my end 

<img width="564" alt="image" src="https://github.com/AY2324S2-CS2103T-T11-4/tp/assets/114789615/46eb8587-c606-41b9-bbc7-5c0cd0562d65">
<img width="274" alt="Screenshot 2024-04-03 at 4 47 37 PM" src="https://github.com/AY2324S2-CS2103T-T11-4/tp/assets/114789615/0c8d41d9-a6e1-4341-b87b-6187841e2751">

Reply:
Just noticed that you are reusing the HelpWindow.css here. I think it will be good if you make the style of the buttons consistent with the "Copy URL" button from the help window.